packages/playground/src/portal/dao_view.vue

```diff
@@ -0,0 +1,338 @@
+
+          <v-card-actions>
+            <v-spacer></v-spacer>
+            <v-btn @click="openInfoModal = false"> Close </v-btn>
+          </v-card-actions>
+        </v-card>
+      </v-dialog>
+    </div>
+  </v-container>
+</template>
+<script lang="ts" setup>
+import type { Proposal, Proposals } from "@threefold/tfchain_client";
+import type moment from "moment";
+import { createToast } from "mosha-vue-toastify";
+import { onMounted, ref } from "vue";
+
+import { useProfileManager } from "../stores";
+import type { Farm } from "../types";
+import { getFarms } from "../utils/get_farms";
+import { getGrid } from "../utils/grid";
+
+const loadingProposals = ref(true);
+const activeTab = ref(0);
+const activeProposals = ref<Proposal[]>();
+const inactiveProposals = ref<Proposal[]>();
+const proposals = ref<Proposals>();
+const searchTerm = ref("");
+const openInfoModal = ref(false);
+const openVDialog = ref(false);
+const castedVote = ref(false);
+const loadingVote = ref(false);
+const isValidFarm = ref(false);
+const selectedProposal = ref("");
+const selectedFarm = ref();
+const userFarms = ref<Farm[]>();
+
+const profileManager = useProfileManager();
+const profile = ref(profileManager.profile!);
+const tabs = [
+  { title: "Active", content: activeProposals },
+  { title: "Executable", content: inactiveProposals },
+];
+
+onMounted(async () => {
+  const grid = await getGrid(profile.value);
+
+  if (grid) {
+    proposals.value = await grid?.dao.get();
+    activeProposals.value = proposals.value.active.filter(proposal => proposal.hash);
+    inactiveProposals.value = proposals.value.inactive.filter(proposal => proposal.hash);
+
+    userFarms.value = await getFarms(grid, { ownedBy: profile.value.twinId }, {});
+    loadingProposals.value = false;
+  }
+});
+function expired(proposalEnd: moment.Moment) {
+  return proposalEnd.isAfter(Date.now());
+}
+function openVoteDialog(hash: any, vote: boolean) {
+  openVDialog.value = true;
+  castedVote.value = vote;
+  selectedProposal.value = hash;
+}
+function filteredProposals(proposals: Proposal[] | undefined) {
+  if (searchTerm.value.length) {
+    if (proposals) {
+      return proposals.filter(proposal => proposal.description.includes(searchTerm.value));
+    }
+  }
+  return proposals;
+}
+async function castVote() {
+  loadingVote.value = true;
+  const grid = await getGrid(profile.value);
+
+  if (grid) {
+    try {
+      await grid.dao.vote({
+        address: profile.value.address,
+        farmId: selectedFarm.value,
+        approve: castedVote.value,
+        hash: selectedProposal.value,
+      });
+      createToast("Voted for proposal!", {
+        position: "top-right",
+        hideProgressBar: true,
+        toastBackgroundColor: "green",
+        timeout: 5000,
+        showIcon: true,
+      });
+    } catch (err) {
+      createToast(`Vote Failed!`, {
+        position: "top-right",
+        hideProgressBar: true,
+        toastBackgroundColor: "red",
```

Comment:
Also the color here needs to be changed.
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/40770501/c07a9c5d-bea9-4234-8eef-a2ffabdd624e)